src/raftstore/store/snap.rs

```diff
@@ -447,26 +447,26 @@ impl Snap {
     ) -> RaftStoreResult<Snap> {
         let mut s = Snap::new(dir, key, size_track, false, false, deleter, limiter)?;
         s.set_snapshot_meta(snapshot_meta)?;
-
         if s.exists() {
             return Ok(s);
         }
+
+        let create_new_file = |path: &PathBuf, typ: &str| {
+            let f = OpenOptions::new().write(true).create_new(true).open(path);
+            f.map_err(|e| {
+                error!("tmp {} file exists at {:?}", typ, path);
```

Comment:
Warn is enough. Why not log the path directly?

Reply:
Maybe `tmp cf file exists at /path/to/tmp/cf/files` is more friendly for debug?